src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -332,7 +332,7 @@ public function value($column)
      *
      * @param  int  $count
      * @param  callable  $callback
-     * @return bool
+     * @return  bool
```

Comment:
I wrote `@return_type`, one space!

(also you may want to stash the commits to not clutter git history)